packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -99,6 +103,24 @@ function formatLogEvent(logEvent: LiveTailSessionLogEvent): string {
     return line
 }
 
+//Auto scroll visible LiveTail session editors if the end-of-file is in view.
+//This allows for newly added log events to stay in view.
+function getTextEditorsToScroll(document: vscode.TextDocument): vscode.TextEditor[] {
+    return vscode.window.visibleTextEditors.filter((editor) => {
```

Comment:
If someone has a two visible editors, one on the left and one on the right like:
```
____________________________________________________
|                      |                           |
| first editor         |       second editor       |
|                      |                           |
____________________________________________________
```
does that mean both of them will scroll?

Reply:
Yep! they'd both be considered visible and able to be scrolled if at EOF. If one isn't at EOF, and the other is, then only one would be scrolled.